src/mockRecipes.js

```diff
@@ -1,24 +1,25 @@
-let mockApartments = [
+let mockRecipes = [
   {
-    recipe_name: "Spaghetti Carbonara",
-    description: "A classic Italian pasta dish with creamy sauce.",
-    ingredients: ["Spaghetti", "Eggs", "Pancetta", "Parmesan Cheese", "Black Pepper"],
-    instructions: "Cook spaghetti according to package instructions. In a separate pan, cook pancetta until crispy. In a bowl, whisk eggs, grated Parmesan cheese, and black pepper. Once spaghetti is cooked, drain and add to the pan with pancetta. Turn off heat, add egg mixture, and toss until coated and creamy. Serve immediately.",
+    recipe_name: "Grilled Salmon with Asparagus",
+    description: "A delicious and healthy dish featuring grilled salmon fillets served with roasted asparagus.",
+    ingredients: ["Salmon fillets", "Asparagus", "Olive oil", "Lemon", "Garlic", "Salt", "Black pepper"],
```

Comment:
Make sure your data type in your mockRecipes match the schema on the backend. As I recall, setting up ingredients to be an array was a stretch goal.